integration-tests/actions/vrfv2_actions/vrfv2_steps.go

```diff
@@ -308,6 +350,80 @@ func SetupVRFV2Environment(
 	return vrfv2Contracts, subIDs, &data, nil
 }
 
+func SetupVRFV2WrapperEnvironment(
+	env *test_env.CLClusterTestEnv,
+	vrfv2Config vrfv2_config.VRFV2Config,
+	linkToken contracts.LinkToken,
+	mockNativeLINKFeed contracts.MockETHLINKFeed,
+	coordinator contracts.VRFCoordinatorV2,
+	keyHash [32]byte,
+	wrapperConsumerContractsAmount int,
+) (*VRFV2WrapperContracts, *uint64, error) {
+	// Deploy VRF v2 direct funding contracts
+	wrapperContracts, err := DeployVRFV2DirectFundingContracts(
+		env.ContractDeployer,
+		env.EVMClient,
+		linkToken.Address(),
+		mockNativeLINKFeed.Address(),
+		coordinator,
+		wrapperConsumerContractsAmount,
+	)
+	if err != nil {
+		return nil, nil, err
+	}
+	err = env.EVMClient.WaitForEvents()
+	if err != nil {
+		return nil, nil, errors.Wrap(err, ErrWaitTXsComplete)
+	}
+
+	// Configure VRF v2 wrapper contract
+	err = wrapperContracts.VRFV2Wrapper.SetConfig(
+		vrfv2Config.WrapperGasOverhead,
+		vrfv2Config.CoordinatorGasOverhead,
+		vrfv2Config.WrapperPremiumPercentage,
+		keyHash,
+		vrfv2Config.WrapperMaxNumberOfWords,
+	)
+	if err != nil {
+		return nil, nil, err
+	}
+	err = env.EVMClient.WaitForEvents()
+	if err != nil {
+		return nil, nil, errors.Wrap(err, ErrWaitTXsComplete)
+	}
```

Comment:
checked with Test Tooling team - we should use go's standard package for dealing with errors instead of _"cosmossdk.io/errors"_ package

`fmt.Errorf("%s: %w", "msg", err)`

e.g. [link](https://github.com/smartcontractkit/chainlink/blob/develop/integration-tests/actions/vrfv2plus/vrfv2plus_steps.go#L704C1-L705C1)